Sources/StreamChat/Controllers/BackgroundListDatabaseObserver.swift

```diff
@@ -177,8 +187,6 @@ class BackgroundListDatabaseObserver<Item, DTO: NSManagedObject> {
         guard !isInitialized else { return }
         isInitialized = true
 
-        onWillChange?()
```

Comment:
why is this not needed anymore?

Reply:
I'm a bit concerned about this one 🤔 Won't this have any breaking behaviour?

Reply:
It was a mistake on the previous implementation of [BackgroundListDatabaseObserver.swift](https://github.com/GetStream/stream-chat-swift/pull/2831/files/54cb84326cd4a4bcd9c1a2a79babdd5c133ac9b2#diff-f87ca7837e4104c46b1b50f6256905f9120d826b2962fa91e6d4a0a8b7557e03)